src/rest_framework_dso/renderers.py

```diff
@@ -272,6 +272,12 @@ def _get_csv_header(self, serializer: Serializer):
     def render(self, data, media_type=None, renderer_context=None):
         if (serializer := get_data_serializer(data)) is not None:
             header, labels = self._get_csv_header(serializer)
+
+            # Ensure fields are ordered the way they were requested.
+            request_fields = renderer_context["request"].GET.get("_fields")
+            if request_fields:
+                header = request_fields.split(",")
+
```

Comment:
Hmm dit overruled dus compleet `_get_csv_header()`. Kunnen we deze code niet beter daar neerzetten?